### PR TITLE
fix: added url and token fields for cluster

### PR DESCRIPTION
**Pull request by edewit**
fixing: https://github.com/fabric8-launcher/launcher-application/issues/896

### CODE REVIEW

**Comment by fabric8cd** [comment]
This PR was deployed **with mocked data** in the following URLs for your appreciation:
- Storybook: https://fabric8-launcher-frontend-pr-1325-storybook.surge.sh

**Comment by edewit** [comment]
@ia3andy can you have a look?

File: packages/launcher-client/src/client/impl/default.launcher.client.ts [hunk 1]
```diff
@@ -204,6 +206,12 @@ export default class DefaultLauncherClient implements LauncherClient {
     if (config.clusterId) {
       headers['X-OpenShift-Cluster'] = config.clusterId;
     }
+    if (config.clusterUrl) {
+      headers['X-OpenShift-Cluster-URL'] = config.clusterUrl;
+    }
+    if (config.clusterToken) {
+      headers['X-OpenShift-Authorization'] = config.clusterUrl;
```
Comment (gastaldi) [inline packages/launcher-client/src/client/impl/default.launcher.client.ts]: This looks wrong to me. It should always be a JWT token

Comment (gastaldi) [inline packages/launcher-client/src/client/impl/default.launcher.client.ts]:
```suggestion
      headers['X-OpenShift-Authorization'] = config.clusterToken;
```

Comment (edewit) [reply packages/launcher-client/src/client/impl/default.launcher.client.ts]: good catch, that was definitely wrong

**Pull request closed without merging**